cmd/notation/cert_gen.go

```diff
@@ -43,14 +43,14 @@ func generateTestCert(opts *certGenerateTestOpts) error {
 	fmt.Println("generated certificates expiring on", rsaLeafCertTuple.Cert.NotAfter.Format(time.RFC3339))
 
 	// write private key
-	keyPath := dir.Path.Localkey(name, dir.KeyExtension)
+	keyPath := dir.Path.Localkey(name, dir.LocalKeyExtension)
```

Comment:
Update this according to https://github.com/notaryproject/notation-go/pull/104?

Reply:
Updated.